html/browsers/browsing-the-web/unloading-documents/support/001-1.html

```diff
@@ -13,7 +13,7 @@
  })
 onload = t.step_func(function() {
   localStorage.test6564729 = '0';
-  setTimeout(t.step_func(function() {document.links[0].click()}));
+  step_timeout(t.step_func(function() {document.links[0].click()}));
```

Comment:
This should instead replace the t.step_func call with a t.step_timeout call and then fixup the parenthesis.